docs/omero_figure_scripting.rst

```diff
@@ -12,16 +12,41 @@ In this example, we will automatically add labels based on analysis results.
 ----------
 
 -  Install the OMERO.figure web app as described at https://pypi.org/project/omero-figure/
+-  Upload the script at ../scripts/Split_View_Figure.py to the OMERO scripting service
 
 **Resources:**
 --------------
 
--  2 FRAP images from https://downloads.openmicroscopy.org/images/DV/will/FRAP/
+-  Any multi-channel images, such as https://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
+-  Any time-lapse images, such as https://downloads.openmicroscopy.org/images/DV/will/FRAP/
 
 **Step-by-Step:**
 -----------------
 
-We will use the images listed above to create a FRAP figure but you can use any time-lapse images.
+Figure creation in Python
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OMERO.figure files are simply JSON data, stored in OMERO File Annotations with a specific
+namespace of omero.web.figure.json. We can create these files using Python scripts, uploaded to
+the OMERO.scripting service to make them available to all OMERO users.
+
+The format of the JSON is described at https://github.com/will-moore/figure/blob/figure_json_docs/docs/figure_file_format.rst.
```

Comment:
This will need to be updated

Reply:
Need https://github.com/ome/training-scripts/pull/86 to be merged first

Reply:
PR has now been merged.

Reply:
Thanks. Link updated.